### DISCUSSION TITLE
What is your favorite Pokémon?

### DISCUSSION THREAD

**Discussion by Marsall-k** [NAIST-SE/AttractingContributors#1176] [2024 Attraction Contributions]
Please tell me your favorite Pokémon! I like Slowpoke.
好きなポケモンを教えてください！僕はヤドンが好きです．

https://github.com/Marsall-k/AttractingContributors-Pokemon

**Comment by honamikoyama** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
https://github.com/honamikoyama/Travel.git
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1182

**Comment by marshr00m** [comment]
I submitted a pull request. I would appreciate any additions to my list!
https://github.com/marshr00m/AttractingContributors-Where-would-you-like-to-travel
